Dockerfile.in

```diff
@@ -1,11 +1,12 @@
 FROM alpine/helm:VERSION
 ENV HOME /tmp
 
-#FIXME: add versioning to helm-s3
```

Comment:
Please preserve the fixme unless you're going to fix it.

Reply:
done !!